integration-cli/docker_cli_checkpoint_test.go

```diff
@@ -0,0 +1,39 @@
+// +build experimental
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/go-check/check"
+)
+
```

Comment:
Can we have a bit more complicated tests. Something that shows that the state is preserved and something that uses the custom locations and force.